### DISCUSSION TITLE
LAN, RS questions

### DISCUSSION THREAD

**Discussion by maxtattonbrown** [StealthChesnut/HA-FoxESS-Modbus#105] [Q&A]
Hi there,

First of all, great project.

Secondly, I'm on old enough firmware to work over standard ethernet and it seems to not be working. I don't even get a PV_daily figure to add for the energy dashboard. Any ideas?

Lastly, my RS port seems to already have something plugged into it, with a wire zooming off into some trunking. Any idea what that would be?

Many thanks,

Max

**Comment by StealthChesnut** [comment]
First - I would make sure you're using the LAN config file, not the USB config file. Can you ping the inverter IP? Make sure you've got network connectivity first, and then go from there. Also check your HA logs for errors/warnings.

The communications port also likely has your CT clamp wired into it. If the wire goes off to your elec meter, and you've got a small box clamped around the live feed cable it's your CT clamp.
